pmd-java/src/main/resources/category/java/performance.xml

```diff
@@ -369,15 +369,29 @@ buf.append("1m");           // good
 String.trim().length() is an inefficient way to check if a String is really empty, as it
```

Comment:
I'd probably also mention `String.trim().isEmpty()` which this rule also detects... the whole point is don't use `trim()` (which creates a new string) when you just need a boolean check